### PR TITLE
Edit OpenGov Delegation Info

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK video tutorials are ready, but I am waiting for Annika. I will add them later on.

File: docs/learn/learn-guides-polkadot-opengov.md [hunk 1]
```diff
@@ -13,6 +13,105 @@ This page is for advanced users of Polkadot OpenGov. If you would learn about an
...
+You can start delegating your votes by clicking the "Delegate" button on
+[Governance > Referenda](https://polkadot.js.org/apps/#/referenda).
+
+![js-delegation-start](../assets/js-delegation-start.png)
+
+If it is the first time you delegate or vote, there will be a banner message. You can delegate
+single tracks or all the tracks. You can specify the number of votes (i.e., the number of tokens)
+and the [conviction multiplier](./learn-polkadot-opengov.md#voluntary-locking). After clicking
+"Next", you will need to specify the account to delegate your votes to, and after clicking
+"Delegate" and "Sign and Submit" your delegations will appear for each track (see below).
```
Comment (profradha) [inline docs/learn/learn-guides-polkadot-opengov.md]:
```suggestion
If it is the first time you delegate or vote, there will be a banner message. You can delegate on a
single track or all the tracks. You have an option to specify the number of votes (i.e., the number of tokens)
and the [conviction multiplier](./learn-polkadot-opengov.md#voluntary-locking). After clicking
"Next", you will need to specify the account to delegate your votes to, and after clicking
"Delegate" and "Sign and Submit" your delegations will appear for each track (see below).
```

File: docs/learn/learn-guides-polkadot-opengov.md [hunk 2]
```diff
@@ -13,6 +13,105 @@ This page is for advanced users of Polkadot OpenGov. If you would learn about an
...
+
+![js-delegation-allTracks](../assets/js-delegation-allTracks.png)
+
+Note that if you want to delegate just a few tracks, you have two options:
+
+- Repeat the process through the "Delegate" tab multiple times
+- Issues a batch call with multiple `convictionVoting.delegate` extrinsics under
+  [Developer > Extrinsics](https://polkadot.js.org/apps/#/extrinsics)
```
Comment (profradha) [inline docs/learn/learn-guides-polkadot-opengov.md]:
```suggestion
- Issue a batch call with multiple `convictionVoting.delegate` extrinsics under
  [Developer > Extrinsics](https://polkadot.js.org/apps/#/extrinsics)
```

File: docs/learn/learn-guides-polkadot-opengov.md [hunk 3]
```diff
@@ -13,6 +13,105 @@ This page is for advanced users of Polkadot OpenGov. If you would learn about an
...
+
+![js-delegation-delegate](../assets/js-delegation-delegate.png)
+
+By clicking on "Add item" you can add new extrinsics for multiple tracks.
+
+### Undelegate Votes
+
+The "Delegate" button on [Governance > Referenda](https://polkadot.js.org/apps/#/referenda) is only
+for delegate votes. You cannot undelegate or modify your delegations. If you wish to undelegate, you
```
Comment (profradha) [inline docs/learn/learn-guides-polkadot-opengov.md]:
```suggestion
for delegating votes. You cannot undelegate or modify your delegations. If you wish to undelegate, you
```

File: docs/learn/learn-guides-polkadot-opengov.md [hunk 4]
```diff
@@ -13,6 +13,105 @@ This page is for advanced users of Polkadot OpenGov. If you would learn about an
...
+[Developer > Extrinsics](https://polkadot.js.org/apps/#/extrinsics) and submit a
+`convictionVoting.unlock` extrinsic to unlock funds for a specific track. Note that if you delegated
+multiple tracks, the funds will be unlocked after undelegating all the tracks.
+
+![js-delegation-unlock](../assets/js-delegation-unlock.png)
+
+### Modify your Delegations
+
+The "Delegate" on [Governance > Referenda](https://polkadot.js.org/apps/#/referenda) is only for
+delegate votes. You cannot undelegate or modify your delegations. If you wish to update the
```
Comment (profradha) [inline docs/learn/learn-guides-polkadot-opengov.md]:
```suggestion
delegating votes. You cannot undelegate or modify your delegations. If you wish to update the
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**